src/reanimated2/screenTransition/swipeSimulator.ts

```diff
@@ -0,0 +1,228 @@
+'use strict';
+import type {
+  PanGestureHandlerEventPayload,
+  ScreenTransitionConfig,
+  RNScreensTurboModuleType,
+} from './commonTypes';
+import { applyStyle } from './styleUpdater';
+
+const RNScreensTurboModule: RNScreensTurboModuleType =
+  global.RNScreensTurboModule || {
+    startTransition: (_stackTag) => {
+      'worklet';
+      console.warn('[Reanimated] RNScreensTurboModule has not been found.');
+      return {
+        topScreenId: -1,
+        belowTopScreenId: -1,
+        canStartTransition: false,
+      };
+    },
+    updateTransition: (_stackTag, _progress) => {
+      'worklet';
+      console.warn('[Reanimated] RNScreensTurboModule has not been found.');
+    },
+    finishTransition: (_stackTag, _isCanceled) => {
+      'worklet';
+      console.warn('[Reanimated] RNScreensTurboModule has not been found.');
+    },
```

Comment:
Maybe we move the creation of fallback object to some factory function in order to decrease the density of code here?